<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>660</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>221</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="x0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>X0</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="x0_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="y0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Y0</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="y0_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="xe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Xe</string>
    </property>
   </widget>
   <widget class="QLabel" name="ye">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ye</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="xe_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="ye_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="line_interpolation">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>97</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>直线插补</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arc_interpolation">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>97</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>圆弧插补</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="R_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="r">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="r_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>R&gt;0则进行顺时针插补，R&lt;0进行逆时针插补</string>
    </property>
   </widget>
   <widget class="QSlider" name="speed">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="r_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>68</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>慢</string>
    </property>
   </widget>
   <widget class="QLabel" name="r_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>68</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>快</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuX0">
    <property name="title">
     <string>X0</string>
    </property>
   </widget>
   <addaction name="menuX0"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
